src/plugins/plugins.js

```diff
@@ -182,6 +184,7 @@ define([
     plugins.GoToOriginalAction = GoToOriginalAction.default;
     plugins.ClearData = ClearData;
     plugins.WebPage = WebPagePlugin.default;
+    plugins.Condition = ConditionPlugin.default;
```

Comment:
Why does this assignment use "default"? 
Should probably just be `plugins.ConditionSet = ConditionSetPlugin`

Reply:
I based this on WebPage and it just worked. If I remove the ".default" it breaks. Not sure why that's not the case with ClearData and several others. For now I'll leave it as is.

Reply:
@JoelMcKinnon @shefalijoshi Those plugins use ES6 export default and that's why `.default` is needed. The other plugins use ReuireJS for loading modules.

Reply:
OK! thanks.